.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore bak file
+**~
+
+# Ignore idea project metadata
+.idea/
```

Comment:
sure we can add a lot of things here, I have not found the need to do that yet... Not sure what would be the best approach here: add things as they appear of preempt?
In the seba project we have :   

# Byte-compiled / optimized / DLL files
__pycache__/
*.py[cod]

# C extensions
*.so

# Distribution / packaging
.Python
env/
build/
develop-eggs/
dist/
downloads/
eggs/
lib/
lib64/
parts/
sdist/
var/
*.egg-info/
.installed.cfg
*.egg

# PyInstaller
#  Usually these files are written by a python script from a template
#  before PyInstaller builds the exe, so as to inject date/other infos into it.
*.manifest
*.spec

# Installer logs
pip-log.txt
pip-delete-this-directory.txt

# Unit test / coverage reports
htmlcov/
.tox/
.coverage
.cache
nosetests.xml
coverage.xml

# Translations
*.mo
*.pot

# Django stuff:
*.log

# Sphinx documentation
docs/_build/

# PyBuilder
target/

# Backup file
*~

# Idea files
.idea